src/app/components/landingRight.ts

```diff
@@ -129,17 +131,18 @@ const landingRight = {
       this.showErrors = !this.showErrors
     },
     signupValid() {
-      const _emailRegix = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
-      return _emailRegix.test(String(this.signup.email).toLowerCase());
+      // eslint-disable-next-line max-len
+      const _emailRegix = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+      return _emailRegix.test(String(this.signup.email).toLowerCase())
     },
-    signupNameValid() {
+    signupNameValid(): boolean {
       return this.signup.accountName !== null
     },
     pantryIDValid() {
       return this.pantry.id !== null
     },
-    createAccountButtonDisabled() {
-      return !this.signupNameValid() || this.accountCreationInProgress
+    createAccountButtonDisabled(): boolean {
+      return !this.signupNameValid() || !this.accountCreationInProgress
```

Comment:
Not sure if this was intended but prefixing `!` to `this.accountCreationInProgress` changes this method's behaviour.

Reply:
Nope! This was a typo. Fixed it now.